unlock-app/src/utils/checkoutLockUtils.ts

```diff
@@ -99,10 +99,9 @@ export const formattedKeyPrice = (
     return 'FREE'
   }
   const price = keyPrice ? parseFloat(keyPrice) * numberOfRecipients : null
-  return `${price?.toString().substring(0, 9)} ${lockTickerSymbol(
-    lock,
-    baseCurrencySymbol
-  )}`
+  return `${price
```

Comment:
keeping `.substring(0, 9)` here will cut the price is too long and make it not understandable

Reply:
+1